tests/handler/handler_copy.cpp

```diff
@@ -6,6 +6,11 @@
 //
 *******************************************************************************/
 
+#include <regex>
+#include <mutex>
+#include <sstream>
+#include <memory>
```

Comment:
While I don't disagree, whenever a project provides a `.clang-format` file I just assume that everything is in order and don't manually check for things like this. Since I see you making this review on pretty much every PR, maybe you should consider setting [`SortIncludes: true`](https://github.com/KhronosGroup/SYCL-CTS/blob/26f29d2c8010125a85e8b215172a17f40424c8e8/.clang-format#L61)? :)

Reply:
Interesting point!
I guess you assume that everybody or some CI uses `clang-format`. :-)
It was added by @Ruyk on internal GitLab commit 42ee2db00faf8d31f2b665134312bd0092a2d87f Date:   Wed Aug 23 16:16:12 2017 +0100 and never been updated since.
Perhaps we have just discovered our new `clang-format` champion with this PR? :-)
Feel free to suggest a lot of improvements in this area.

Reply:
> While I don't disagree, whenever a project provides a `.clang-format` file I just assume that everything is in order and don't manually check for things like this. Since I see you making this review on pretty much every PR, maybe you should consider setting [`SortIncludes: true`](https://github.com/KhronosGroup/SYCL-CTS/blob/26f29d2c8010125a85e8b215172a17f40424c8e8/.clang-format#L61)? :)

Feel free to change it. :-)
I cannot believe that our `.clang-format` is so complex and different from a standard one...